samples/apps/hugging-face-http-server/README.md

```diff
@@ -1,4 +1,4 @@
-# Local Hugging Face Model Inteference Server
+# Local Hugging Face Model Interference Server
```

Comment:
Hmm, I thought it was a single typo, but now I realize there are many unrelated to the PR
```
Warning: "ags" should be "ages" or "tags".
Warning: "FUNCTIO" should be "FUNCTION".
Warning: "sesion" should be "session".
Warning: "sesion" should be "session".
Warning: "caf" should be "calf".
Warning: "caf" should be "calf".
Warning: "Iteraction" should be "Interaction" or "Iteration".
Warning: "Iteraction" should be "Interaction" or "Iteration".
Warning: "empted" should be "emptied".
```
It says "Warning" but yet exited with Error code...
Maybe we need a second PR which fixes all the typos before this can go in 😟

Reply:
#1377 has fixed the existing problems that popped up (stil not sure why suddenly).

Reply:
Sorry for the hassle @mattmazzola  -- can just stick with your original changes now.

Reply:
Ok, yea. Updated with main and reverted this change